samples/ResourceGroup/copy-resourcegroup-tag/azurepolicy.json

```diff
@@ -2,18 +2,26 @@
     "properties": {
         "displayName": "Copy resource group tag to resource",
         "mode": "Indexed",
-        "description": "Copy a specific tag from the resource group to the resource.  For example, a tag named example on the resource group will be copied to the resources.  Change the word example in the policy rule to change the name of the tag copied to a resource.",
+        "description": "Copy a tag specified in the parameter value from the resource group to the resource.  For example, a tag named costCenter on the resource group will be copied to the resources.",
+        "parameters": {
+            "tagName": {
+                "type": "String",
+                "metadata": {
+                    "description": "Name of the tag, such as costCenter"
+                }
+            }
+        },
         "policyRule": {
             "if": {
-                "field": "tags.example",
+                "field": "[concat('tags.', parameters('tagName'))]",
```

Comment:
>"field": "[concat('tags.', parameters('tagName'))]", [](start = 16, length = 52)

Should use bracket syntax to allow wider variety of parameter values (e.g. tag names with dashes in them)

`"field": "[concat('tags[', parameters('tagName'), ']')]",`